README.md

```diff
@@ -565,6 +570,68 @@ this will evaluate this model across all datapoints in the `data/combined.pkl.gz
 which contains data from the `train`, `val`, and `test` sets so that
 evaluation doesn't have to be run on each set separately.
 
+### 🏘 ProcTHOR pre-training
+
+We include commands that can be used to generate a ProcTHOR-pretrained agent and
+then fine-tune it with the 2022 rearrange dataset. Please note that this only covers the 1-phase modality,
+for which we also provide a
+[pre-trained and fine-tuned checkpoint](https://prior-model-weights.s3.us-east-2.amazonaws.com/embodied-ai/rearrangement/one-phase/exp_iThorOnePhaseRGBClipResNet50FineTune_procthor180Msteps_ithor_splits_ithor_fine_tune_64_to_128_rollout_3Msteps_6Msteps__stage_02__steps_000016018675.pt).
+We also provide scripts to generate new ProcTHOR datasets in case you want to try new episodes. 
+
+#### Pre-train model in ProcTHOR (single machine)
+The following will take for about 10-14 days on an 8-GPU machine with 56 CPU cores:
+```bash
+allenact -b baseline_configs/one_phase/procthor one_phase_rgb_clip_dagger \
+ -s 12345 --config_kwargs '{"distributed_nodes":1}' 
+```
+We **strongly** recommend using a larger number of GPUs and computing nodes for this step.
+
+#### ProcTHOR mini-valid
+Run ProcTHOR mini-valid on checkpoints under a `CKPT_DIR` directory:
+```bash
+inv make-valid-houses-file
+allenact -b baseline_configs/one_phase/procthor/eval eval_minivalid_procthor \
+ -s 12345 --eval --approx_ckpt_step_interval 5e6 -c CKPT_DIR
+```
+
+#### Fine-tune model in iTHOR (single machine)
+
+The following will take for about two days on an 8-GPU machine with 56 CPU cores.
+Assuming the chosen checkpoint from ProcTHOR pre-training has path `CKPT_PATH`:
+```bash
+allenact -b baseline_configs/one_phase/procthor/ithor ithor_one_phase_rgb_fine_tune \
+ -s 12345 -c CKPT_PATH --restart_pipeline
+```
+
+#### iTHOR mini validation
+Run iTHOR mini-valid on checkpoints under `CKPT_DIR`:
+```bash
+inv make-ithor-mini-val
+allenact -b baseline_configs/one_phase/procthor/eval eval_minivalid_ithor \
+ -s 12345 --eval --approx_ckpt_step_interval 5e6 -c CKPT_DIR
+```
+
+#### Generate new ProcTHOR training and mini-valid episodes
+Our training and validation datasets are already provided under [data/2022procthor](data/2022procthor), but we also provide our used
+scripts in case any user is interested in trying new episode distributions.
+
+For training, we use a dataset composed of 50,000 episodes sampled from 2,500 houses with one or two rooms.
+The following commands take for several hours on an 8-GPU machine with 56 CPU cores:
+```bash
+python datagen/procthor_datagen/datagen_runner_train.py
+inv make-procthor-mini-train
+```
+
+To create a ProcTHOR mini-valid dataset, the following commands take for several hours on an 8-GPU machine
+with 56 CPU cores: 
+```bash
+python datagen/procthor_datagen/datagen_runner_valid.py
+inv consolidate-procthor-val
+inv make-procthor-mini-val
+inv make-valid-houses-file
+```
+
+
```

Comment:
It makes sense. I might have missed it, but I didn't see the citation for EmbCLIP and I thought it would be best to not add it for consistency. In any case, I'm happy to add it 👍